### PR TITLE
Fish animation based on dev branch

### PR DESCRIPTION
**Pull request by bingxiaan**
Fish animation based on dev branch

### CODE REVIEW

File: app/src/main/java/com/hydrofish/app/ui/composables/tabs/HomeScreen.kt [hunk 1]
```diff
@@ -140,3 +173,319 @@ fun ReusableDrinkButton(waterAmt: Float, hydroFishViewModel: HydroFishViewModel
...
+    val fish_v4 = PathParser().parsePathString(
+        "M2,2C2,10.323 5,18.017 5,26.5C5,33.508 8,40.756 8,48"
+    ).toNodes()
+    val fish_v5 = PathParser().parsePathString(
+        "M2,2C2,3.7633 3.5079,3.754 2,3"
+    ).toNodes()
+}
+
+
+object SharkPaths {
```
Comment (KevinZxy1273) [inline app/src/main/java/com/hydrofish/app/ui/composables/tabs/HomeScreen.kt]: Please move the image motion paths to a separate file to keep HomeScreen.kt clean and readable.

**Review by KevinZxy1273 — CHANGES_REQUESTED** [review]
Reviewed and looks good, but please move image motion paths to a separate file for cleaner HomeScreen.kt. Thanks!

File: app/src/main/java/com/hydrofish/app/ui/composables/tabs/HomeScreen.kt (hunk 1 above)
Comment (bingxiaan) [reply app/src/main/java/com/hydrofish/app/ui/composables/tabs/HomeScreen.kt]: Finished moving paths.

**Review by isabellachen3988 — APPROVED** [review]
all good

**Review by KevinZxy1273 — APPROVED** [review]

**Review by marklhj-lee — APPROVED** [review]

**Pull request merged**